src/frontend/js/widgets/Dashboard/components/TeacherProfileDashboardSidebar/components/OrganizationLinks/_styles.scss

```
@@ -1,7 +1,7 @@
 .dashboard-sidebar {
   &__organization-section {
-    // padding: rem-calc(5px) rem-calc(8px) rem-calc(40px);
-    padding: rem-calc(20px) rem-calc(8px) rem-calc(40px);
+    padding: rem-calc(20px) rem-calc(8px) 0
+      rem-calc(r-theme-val(dashboard-sidebar, menu-link-left-spacing));
```

Little suggestion to improve readability : 
```suggestion
    $padding-left: r-theme-val(dashboard-sidebar, menu-link-left-padding); 
    padding: rem-calc(20px) rem-calc(8px) 0 rem-calc($padding-left);
```